<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QKeyboardASCIIGen</class>
 <widget class="QDialog" name="QKeyboardASCIIGen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_btntype">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string> 按钮属性</string>
   </property>
   <widget class="QPushButton" name="State1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>按下后</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>边框色：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>背景色：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>状  态：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>样  式：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="State0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>按下前</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>前景色：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Frame_Color">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="FG_Color">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BG_Color">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="Pattern">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_btntype_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>操作类型</string>
   </property>
   <widget class="QRadioButton" name="ENTER">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ENTER</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="CLEAR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CLR</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="ASCII">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII码</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="ASCIIText">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>1</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="BACKSPACE">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DEL</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="ESC">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ESC</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="CAPSLOCK">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CAPSLOCK</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>541</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>文本属性</string>
   </property>
   <widget class="QComboBox" name="combo_Language">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_type_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>语言</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_Alignment">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>居中</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>左对齐</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>右对齐</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_type_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>对齐</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_type_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>颜色</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_type_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>字体</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_type_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>大小</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_boldcheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>粗体</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_underlinecheck">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>下划线</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_italiccheck">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>斜体</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_font">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="m_textbtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="m_fontsizecmb">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="m_text">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_btntype_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>图片</string>
   </property>
   <widget class="QRadioButton" name="m_sysradio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>系统图库</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_fileradio">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_slectpicbtn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>45</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/standardtool/images/standardtool/open.ico</normaloff>:/standardtool/images/standardtool/open.ico</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>46</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>选择图片：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_deletepicbtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删除图片</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
